### PR TITLE
feat: add send_event/2

### PR DESCRIPTION
**Pull request by vanppo**

### CODE REVIEW

**Review by fahchen — DISMISSED** [review]

File: lib/jet_plugin_sdk/jet_client/jet_client.ex [hunk 1]
```diff
@@ -123,4 +123,33 @@ defmodule JetPluginSDK.JetClient do
         otherwise
     end
   end
+
+  @send_event_query """
+  mutation sendEvent(
```
Comment (CptBreeza) [inline lib/jet_plugin_sdk/jet_client/jet_client.ex:128]: 咱们一般给 query 和 mutation 起的名字都是全大写的：sendEvent -> SendEvent

Comment (vanppo) [reply lib/jet_plugin_sdk/jet_client/jet_client.ex:128]: 没有吧 我看到的都是驼峰呢
https://github.com/Byzanteam/jet-plugin-sdk/blob/90bd7524f58c8c46ac70732594299ec7463e9142/lib/jet_plugin_sdk/jet_client/jet_client.ex#L91-L111

Comment (CptBreeza) [reply lib/jet_plugin_sdk/jet_client/jet_client.ex:128]: 之前写得有点问题。后面就都写成大写吧

Comment (CptBreeza) [reply lib/jet_plugin_sdk/jet_client/jet_client.ex:128]: https://graphql.org/learn/queries/#using-variables-inside-fragments
https://graphql.org/learn/queries/#mutations

Comment (vanppo) [reply lib/jet_plugin_sdk/jet_client/jet_client.ex]: 再看下呢

**Review by CptBreeza — APPROVED** [review]

**Review by fahchen — APPROVED** [review]

**Pull request merged**